kbuild/utils/generate_custom_archive.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# Generate a custom kernel (to build external modules) accordingly to what our
+# bazel kernel build rules expect (i.e., a .tar.gz just like the ones
+# generated by kbuild).
+
+usage()
+{
+        cat <<EOF
+Generate a kbuild compliant .tar.gz starting from a compiled custom linux source tree.
+usage: ${0##*/} OPTS
+    -k KDIR      Directory containing the compiled custom linux source tree
+    -t OUTDIR    Root directory for containing the generated kernel package
+    -v KVER      Kernel version (used to name the package)
+This will generate a KVER-test.tar.gz file inside OUTDIR, built from the pre-compiled kernel in KDIR.
```

Comment:
nit: I would expand the instructions here more. What is your defintion of "built" from the pre-compiled kernel in KDIR?

The script does not invoke any make target, and does not check anything present in the tree. Is `make modules` enough? Can I just unpack the tarball and have a .config file? what is the recommendation?

Reply:
Added the expected steps preceding the call to this script.